### ISSUE TITLE
Pixelated/ Not clear video feed

### ISSUE THREAD

**Issue by BROLLIC** [uavpal/disco4g#31]
Hello, I have successfully installed the softmod on both my disco and bebop 2. I can get a solid blue light connection on both anywhere between 30-90 seconds. (sometimes it wont connect at all... just blinks blue until red then green). I can operate it fine when connected via lte (flight controls) The only problem (MAJOR) is that the live video feed is pixelated. So I cant properly see where Im going. I actually went 100 feet from my US T-mobile tower and tested its speed. (35mbps down, 40mbps up)....so I know I have enough speed. Could it be my modem? I have the 510 Hi Link modem, I cant find any stick mode ones :(. I have also tried to configure the video feed in the parrot app. I have used android and apple devices. They fly perfectly, no lag in controls at all. No matter how high or how low I go, the video is still not viewable. Please help :)
![thumbnail](https://user-images.githubusercontent.com/41168409/43551664-cd564610-95b5-11e8-9dc2-258289fbcba6.jpg)

**Comment by teja235** [comment]
Do you have T-Mobile One+ ? You need to have a hotspot plan on your phone line in order for the controller to get the full bandwidth your phone has. It only costs an extra $10 /month if you already have T-Mobile One. Also the sim in your Huawei must be the same as your phone (Tmobile), and must has a data plan that works for hotspot devices. I have the 10 gb /month plan. Gets me about 11 hours of 4G flight

**Comment by BROLLIC** [comment]
Yes I have all the Tmobile bells and whistles! Ive tested my phones hotspot speeds with 3 other tablets/phones. I can get 25 mbps down and 30 mbps up. So what else could it be :(?

**Comment by teja235** [comment]
Yea, test the speed and latency of your phone hotspot from a different
device. Then test the huawei speed and latency by plugging it into a laptop
or something. If everything checks out, check your settings on zerotier.
Ethernet bridging should be checked.

If all this doesn’t fix the issue, maybe it’s just the quality of signal at
the spot you’re taking off from. Cell signal is just another radio signal
and susceptible to noise from obstacles and reflections. Try a different
spot! I had to try a couple different ones before I could find one with
direct line of sight to a tower and quiet enough to fly without people
asking questions of legality 😅 use the apps Sensorly and OpenSignal to
find your nearest tower and measure the signal and speed.

**Comment by BROLLIC** [comment]
I did all of that lol. I was litterally 100 feet from a tower... If could have sat on it if i wanted to but i thought that might be overkill lol. Still wont work. I tries multiple towers here in NY. Can I purchase a stick mode modem from yall? Ive seen some uavpal signed modems out there and would love to get one as well.

**Comment by teja235** [comment]
If you have the latest version of the hack installed, it shouldn’t matter.
I have an h modem on T-Mobile and I get a great experience. My plane can
even fly in places where my phone doesn’t get signal, presumably because it
flies over obstacles and valleys, getting a better reception. Maybe
T-Mobile New York just can’t handle the load 😓

**Comment by teja235** [comment]
Just have the make sure my phone is in a good spot to get signal and make a
strong hotspot. Also close all the apps you can and let the phone focus on
being a hotspot. I put it up in a tree like an antenna.

**Comment by BROLLIC** [comment]
Ok everyone. I finally called t mobile up. I explained my situation. They told me that I did purchase the one "PLUS" package, but that there was an error on their side on activating it. So now I have full speed data on my modem attatched to the drones.

So if you ever have video issues but can operate the birds fine, hit up your phone service provider and question your internet speeds! Video's from NY coming soon.
